packages/extension_google_sign_in_as_googleapis_auth/example/android/app/src/main/kotlin/io/flutter/plugins/googlesigninexample/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package io.flutter.plugins.googlesigninexample
```

Comment:
Oops, needs a license. (Tool fix PR incoming.)

Reply:
Good catch! I'll regenerate this with `--android-language=java` so the kt file will go away